### PR TITLE
Chore/mer 106 adding SKU

### PR DESCRIPTION
**Pull request by dbelkad**

### CODE REVIEW

File: Helper/Data.php [hunk 1]
```diff
@@ -476,6 +477,7 @@ public function creditRequest($planId, $depositAmount, $email, $quoteId = null)
                 'name'     => 'Shipping & Handling',
                 'quantity' => (int) 1,
                 'price'    => (int) $shipping,
+                'sku'      => 'SHPNG',
```
Comment (nunocosta-divido) [inline Helper/Data.php:480]: I feel this values should be made a constant

File: Helper/Data.php [hunk 2]
```diff
@@ -485,6 +487,7 @@ public function creditRequest($planId, $depositAmount, $email, $quoteId = null)
                 'name'     => 'Discount',
                 'quantity' => (int) 1,
                 'price'    => (int) ($discount * 100),
+                'sku'      => 'DSCNT',
```
Comment (nunocosta-divido) [inline Helper/Data.php:490]: I feel this values should be made a constant

File: Helper/Data.php (hunk 1 above)
Comment (dbelkad) [reply Helper/Data.php]: Thanks for a tip, will amend right away!

File: Helper/Data.php (hunk 2 above)
Comment (dbelkad) [reply Helper/Data.php]: done

**Review by joelbitar — APPROVED** [review]
Good stuff!

**Pull request merged**